templates/vue/src/components/Lightbox/media/H5PMedia/H5PIframe.vue

```diff
@@ -18,7 +18,7 @@
       width="100%"
       frameborder="0"
       :src="node.typeData && node.typeData.mediaURL"
-      :scrolling="type === 'H5P.InteractiveVideo' && 'no'"
+      :scrolling="library === 'H5P.InteractiveVideo'"
```

Comment:
@wynnset What's the rationale for scrolling behaviour in an H5P.InteractiveVideo?